core-services/prow/02_config/openshift/api/_prowconfig.yaml

```diff
@@ -59,7 +59,67 @@ tide:
     - master
     labels:
     - approved
+    - bugzilla/valid-bug
+    - lgtm
+    missingLabels:
+    - backports/unvalidated-commits
+    - bugzilla/invalid-bug
+    - do-not-merge/hold
+    - do-not-merge/invalid-owners-file
+    - do-not-merge/work-in-progress
+    - keep-main-query-separate
+    - needs-rebase
+    repos:
+    - openshift/api
+  - excludedBranches:
```

Comment:
I don't think this needs to be added, there is already a complement query (=the one with `excludedBranches`)